code/game/objects/items/pet_carrier.dm

```diff
@@ -205,4 +232,32 @@
 	greyscale_config_inhand_right = null
 	greyscale_colors = null
 
+/obj/item/pet_carrier/small
```

Comment:
This could probably use a lathe design (like the base pet carrier has /datum/design/pet_carrier)
Would let players (me) get unnamed ones for capturing small bugs as pets (cockroaches)

Reply:
Full-size pet carriers are already craftable, and hold more animal weight anyway, I'd say it's probably fine to leave as-is.